app/_components/ScheduleCallForm.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+
+import { useState } from "react";
+import { PatientFormState } from "../types";
+import Input from "../_atoms/Input";
+import Checkbox from "../_atoms/Checkbox";
+import Button from "../_atoms/Button";
+
+const ScheduleCallForm: React.FC = () => {
+  const [form, setForm] = useState<PatientFormState>({
+    fullName: "",
+    email: "",
+    phone: "",
+    treatment: "",
+  });
+  const [terms, setTerms] = useState<boolean>(false);
+  const [subscribe, setSubscribe] = useState<boolean>(false);
+  const handleOnChange = (e: any, stateName: string) => {
+    setForm((prev: any) => ({ ...prev, [stateName]: e.target.value }));
+  };
+  return (
+    <form className="md:w-[434px] p-7 flex flex-col gap-3 bg-white mx-auto">
+      <Input
+        label="Full Name"
+        value={form?.fullName}
+        onChange={(e) => handleOnChange(e, "fullName")}
+      />
+      <Input
+        label="Email Address"
+        value={form?.email}
+        onChange={(e) => handleOnChange(e, "email")}
+      />
+      <Input
+        label="Phone Number"
+        value={form?.phone}
+        onChange={(e) => handleOnChange(e, "phone")}
+      />
+      <Input
+        label="Treatment ?"
+        value={form?.treatment}
+        onChange={(e) => handleOnChange(e, "treatment")}
+      />
+      <Checkbox
```

Comment:
yapiyi söyle degistirirsek; "  <Checkbox
        label="Accept the registration terms"
        onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
          setTerms(e.target.checked)
        }
        checked={terms}
      />" 1- daha anlasilir ve e parametresine bir tip atamis oluyoruz.. 2- checked propu zaten checkboxta degeri yönettigi icin value'yü kullanmamis oluruz.